{"uuid":"b888a47d-8ed5-4a2c-a764-db5292ab9eb1","historyId":"dbadccf42404a19e36ec2e8cbff7f097","fullName":"com.autotest.testcase.unittest.AddUserApiTest.adduser","labels":[{"name":"package","value":"com.autotest.testcase.unittest.AddUserApiTest"},{"name":"testClass","value":"com.autotest.testcase.unittest.AddUserApiTest"},{"name":"testMethod","value":"adduser"},{"name":"parentSuite","value":"apiAutoTest"},{"name":"suite","value":"test"},{"name":"subSuite","value":"com.autotest.testcase.unittest.AddUserApiTest"},{"name":"host","value":"xiaohe0128"},{"name":"thread","value":"27788@xiaohe0128.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"正反用例"},{"name":"feature","value":"添加用户接口"},{"name":"epic","value":"接口单元测试"}],"links":[],"name":"adduser","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.reflect.InvocationTargetException","trace":"java.lang.reflect.InvocationTargetException\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat com.autotest.testcase.unittest.AddUserApiTest.reflect(AddUserApiTest.java:29)\r\n\tat com.autotest.testcase.unittest.AddUserApiTest.adduser(AddUserApiTest.java:70)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)\r\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:645)\r\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)\r\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:756)\r\n\tat org.testng.TestRunner.run(TestRunner.java:610)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:387)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:289)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1218)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1133)\r\n\tat org.testng.TestNG.run(TestNG.java:1104)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\nCaused by: java.lang.Exception: Failed : HTTP error code : 400\r\n\tat com.autotest.utils.RequestMethod.Post(RequestMethod.java:107)\r\n\t... 30 more\r\n"},"stage":"finished","steps":[],"attachments":[{"name":"请求地址：","source":"6e50890e-fcec-4954-b755-92b5c8577235-attachment.txt","type":"text/plain"},{"name":"请求方法","source":"0c82d972-3e0a-405b-8c27-15af4be4b34d-attachment.txt","type":"text/plain"},{"name":"请求参数","source":"cad02ed8-eaeb-43fb-a2db-30a67c9ef749-attachment.txt","type":"text/plain"},{"name":"请求头:","source":"23031f85-7a81-44d4-83d5-4575b036acd6-attachment.txt","type":"text/plain"}],"parameters":[{"name":"arg3","value":"4"},{"name":"arg2","value":"??"},{"name":"arg4","value":"nanhai"},{"name":"arg1","value":"中文"},{"name":"arg0","value":"A"}],"start":1727181187772,"stop":1727181187787}